CMakeLists.txt

```diff
@@ -7,9 +7,14 @@ CMAKE_MINIMUM_REQUIRED(VERSION 2.8)
 
 PROJECT(OpenTomb)
 
-SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11 -Wall -Wextra -msse3")
+IF( ${CMAKE_COMPILER_IS_GNUCC} )
```

Comment:
Very nice, the double evaluation was the reason, why I was not able to combine both cases.